buildSrc/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ repositories {
 
 dependencies {
     gradleApi()
-    implementation("org.jetbrains.kotlin.jvm", "org.jetbrains.kotlin.jvm.gradle.plugin", "1.7.21")
+    implementation("org.jetbrains.kotlin.jvm", "org.jetbrains.kotlin.jvm.gradle.plugin", "1.9.0")
```

Comment:
Bumped this to fix this error:

<img width="1646" alt="Screenshot 2023-07-17 at 1 12 43 PM" src="https://github.com/JetBrains/Exposed/assets/38375996/e98028be-b33b-4f5c-9503-414f77d3c413">

Reply:
Should we also bump the Kotlin version from 1.7.21 to 1.9.0? I tried it and there were no issues.